consensus/src/main/java/org/apache/iotdb/consensus/ratis/metrics/RatisMetricSet.java

```diff
@@ -18,21 +18,36 @@
  */
 package org.apache.iotdb.consensus.ratis.metrics;
 
+import org.apache.iotdb.common.rpc.thrift.TConsensusGroupType;
 import org.apache.iotdb.metrics.AbstractMetricService;
 import org.apache.iotdb.metrics.metricsets.IMetricSet;
 
 import org.apache.ratis.metrics.MetricRegistries;
 
 public class RatisMetricSet implements IMetricSet {
   private MetricRegistries manager;
+  private String consensusGroupType;
```

Comment:
We can pass the consensusGroupType as a constructor param, instead of setting it manually.

Reply:
Already changed.